src/contentChunk.js

```diff
@@ -3,7 +3,7 @@ import mongodb from 'mongodb'
 import { connectionDefault } from './config'
 
 let bucket
-const getGridFSBucket = () => {
+export const getGridFSBucket = () => {
```

Comment:
I exported it so it can be used in the router and in the koamiddleware (rawBodyReader f(x)) where bucket creation is done. And this will also ensure that we have one bucket instance being used throughout. The insert function cannot be used because it takes the whole payload as an arg. We are getting chunks and streaming them into gridfs.

Reply:
Ok cool that makes sense. Im just not a fan of having a module that does this for us, and then we implement similar code elsewhere. Maybe we should look at refactoring the GridFS stuff a bit more to better align with this work